### DISCUSSION TITLE
오브젝트 원 운동

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#1111] [5. Q&A]
안녕하세요 오브젝트가 기차처럼 원라인으로 반복해서 움직이게 하고 싶은데 이런건 어떻게 하나요??

**Comment by YSHFM** [comment]
https://docs.unity3d.com/ScriptReference/Transform.RotateAround.html

해당 공식문서 참고하시면 도움되실 것 같습니다.

**Reply by yj098** [reply]
감사합니다 ~~:) 아직은 어렵지만 덕분에 도움이 되었습니다😊😊
